lightly/api/api_workflow_compute_worker.py

```diff
@@ -0,0 +1,100 @@
+from typing import Any, Dict, Optional
+
+from lightly.openapi_generated.swagger_client.models.docker_run_scheduled_create_request import DockerRunScheduledCreateRequest
+from lightly.openapi_generated.swagger_client.models.docker_worker_type import DockerWorkerType
+from lightly.openapi_generated.swagger_client.models.docker_worker_config import DockerWorkerConfig
+from lightly.openapi_generated.swagger_client.models.docker_worker_config_create_request import DockerWorkerConfigCreateRequest
+from lightly.openapi_generated.swagger_client.models.docker_run_scheduled_priority import DockerRunScheduledPriority
+from lightly.openapi_generated.swagger_client.models.create_docker_worker_registry_entry_request import CreateDockerWorkerRegistryEntryRequest
+
+
+class _ComputeWorkerMixin:
+
+    def register_compute_worker(self, name: str = 'Default') -> str:
+        """Registers a new compute worker.
+        
+        Args:
+            name:
+                The name of the compute worker.
+        
+        Returns:
+            The id of the newly registered compute worker.
+
+        """
+        request = CreateDockerWorkerRegistryEntryRequest(
+            name=name, 
+            worker_type=DockerWorkerType.FULL
+        )
+        response = self._compute_worker_api.register_docker_worker(request)
+        return response.id
+
+    def delete_compute_worker(self, worker_id: str):
```

Comment:
do we have the functions to implement a getter? could be very helpful when workers are already registered.

Reply:
Added it :)